cre/go/installer/pkg/protoc_gen.go

```diff
@@ -0,0 +1,187 @@
+package pkg
+
+import (
+	_ "embed"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"strings"
+)
+
+var values = Packages{
+	Go:    "github.com/smartcontractkit/chainlink-protos/cre/go/values/pb",
+	Proto: "values/v1/values.proto",
+}
+
+var sdk = Packages{
+	Go:    "github.com/smartcontractkit/chainlink-protos/cre/go/sdk",
+	Proto: "sdk/v1alpha/sdk.proto",
+}
+
+var creMetadata = Packages{
+	Go:    "github.com/smartcontractkit/chainlink-protos/cre/go/tools/generator",
+	Proto: "tools/generator/v1alpha/cre_metadata.proto",
+}
```

Comment:
Slight change, this used to only own installing the values package and let the others for workflow authors to take care of.